nba/config_classes.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel
+
+class DataPaths(BaseModel):
+    data_file: str
+    another_file: str
```

Comment:
Hey Pushpin @ps2016-mt,
I think the config_classes here doesn't match the config.yaml!
In config.yaml we have both class Data and class Model, which should be the same structure